src/modules/item/export.ts

```diff
@@ -118,6 +126,35 @@ function buildItemEntityMetadata(collection: Collection, item: Item): CatalystIt
   }
 }
 
+function buildTPItemEntityMetadata(
+  collection: Collection,
+  item: Item,
+  itemHash: string,
+  tree: MerkleDistributorInfo
+): CatalystItem | CatalystTPItem {
+  const baseEntityData = {
+    id: buildThirdPartyURN(extractThirdPartyId(collection.urn), collection.id, item.id),
+    ...getBaseEntityMetadata(item)
+  }
+  return {
+    merkleProof: getMerkleProof(tree, itemHash, baseEntityData),
+    ...baseEntityData
+  }
```

Comment:
If you want to avoid a double spread you could:
```ts
{
    ...getBaseEntityMetadata(item)
    merkleProof: getMerkleProof(tree, itemHash, baseEntityData),
    id: buildThirdPartyURN(extractThirdPartyId(collection.urn), collection.id, item.id),
}
```

Reply:
The thing is that I need to send the `baseEntityData` as a parameter to `getMerkleProof`, so I need it to be defined first, that's why I declared it outside the returning object.